doc/classes/Window.xml

```diff
@@ -907,6 +908,18 @@
 		</constant>
 	</constants>
 	<theme_items>
+		<theme_item name="decoration_button_hover_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Window button modulation color, when the button is hovered.
+			[b]Note:[/b] Used only when extend to title is active.
```

Comment:
All of these notes should mention the FLAG_EXTEND_TO_TITLE flag directly.

I also am not sure if these should be notes in the first place. Maybe? But they're not notes, or quirks. They are basically required in order for these items to be useful. But there's no exact rule on what should and shouldn't be a note, either.

Reply:
I modified all the new theme elements to removed the note and add a reference to `extend_to_title` and `buttons are rendered by Godot.`